### DISCUSSION TITLE
Expecting value: line 1 column 1 (char 0)

### DISCUSSION THREAD

**Discussion by g8d3** [langgenius/dify#8604] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I am trying to setup firecrawl with my api key and some url inside of github and I get an error as the title.

![image](https://github.com/user-attachments/assets/a094f2e5-4040-47a1-aa57-1734a98fe20b)

I tried with different URLs, same error.

### 2. Additional context or comments

Not needed.

Thanks.

**Comment by tylerdurden4285** [comment]
I also get this error for the "Podcast Generator" plugin tool. 

> Expecting value: line 1 column 1 (char 0)

Did you find any way to fix it? @g8d3
